nominees/viame.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "VIAME",
+  "description": "A DIY AI system for analyzing imagery and video, orginally targeting marine species analysis.",
```

Comment:
@Msgl could you update the DIY abbreviation to the full form, do-it-yourself in the `description` field. Thanks.